### PR TITLE
WRO-8763: Storybook: migrate default samplers from knobs to controls - About -->Item

### PR DESCRIPTION
**Pull request by vladut-clapou-lgp**
### Checklist

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Migrated default Storybook samplers from knobs to controls - About -->Item

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRO-8763

### CODE REVIEW

File: samples/sampler/stories/default/Button.js [hunk 1]
```diff
@@ -1,44 +1,56 @@
...
+};
+
+export default {
+	title: 'Moonstone/Button',
+	component: 'Button'
+};
+
+export const _Button = (args) => (
+	<Button
+		onClick={action('onClick')}
```
Comment (alexandrumorariu) [inline samples/sampler/stories/default/Button.js]: Sort props here.

File: samples/sampler/stories/default/DayPicker.js [hunk 2]
```diff
@@ -1,32 +1,43 @@
...
+};
+
+export const _DayPicker = (args) => (
+	<DayPicker
+		aria-label={args['aria-label']}
+		dayNameLength={args['dayNameLength']}
+		disabled={args['disabled']}
+		everyDayText={args['everyDayText']}
+		everyWeekdayText={args['everyWeekdayText']}
+		everyWeekendText={args['everyWeekendText']}
```
Comment (alexandrumorariu) [inline samples/sampler/stories/default/DayPicker.js:19]: Remove duplicate

File: samples/sampler/stories/default/DayPicker.js [hunk 3]
```diff
@@ -1,32 +1,43 @@
...
+		title={args['title']}
+	/>
+);
+
+boolean('disabled', _DayPicker, DayPicker);
+select('dayNameLength', _DayPicker, ['short', 'medium', 'long', 'full'], DayPicker, 'long');
+text('aria-label', _DayPicker, DayPicker);
+text('everyDayText', _DayPicker, DayPicker);
+text('everyWeekdayText', _DayPicker, DayPicker);
+text('everyWeekendText', _DayPicker, DayPicker);
```
Comment (alexandrumorariu) [inline samples/sampler/stories/default/DayPicker.js:33]: Same here

File: samples/sampler/stories/default/About.js [hunk 4]
```diff
@@ -1,113 +1,119 @@
...
-// 				>
-// 					<b>Knobs</b> tab lets you adjust component properties
-// 				</HintDialog>
-// 			</div>
-// 		)
-// 	);
+import kind from '@enact/core/kind';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean} from '@enact/storybook-utils/addons/controls';
+import PropTypes from 'prop-types';
```
Comment (alexandrumorariu) [inline samples/sampler/stories/default/About.js]: Order imports here and for the rest of the files (as the components are imported from @enact/moonstone i belive you can add them here also before the @enact/storybook ones)

File: samples/sampler/stories/default/DaySelector.js [hunk 5]
```diff
@@ -1,33 +1,38 @@
...
+};
+
+export default {
+	title: 'Moonstone/DaySelector',
+	component: 'DaySelector'
+};
+
+export const _DaySelector = (args) => (
+	<DaySelector
+		disabled={args['disabled']}
```
Comment (alexandrumorariu) [inline samples/sampler/stories/default/DaySelector.js]: Order props

File: samples/sampler/stories/default/ExpandableInput.js [hunk 6]
```diff
@@ -1,42 +1,52 @@
...
+	<ExpandableInput
+		disabled={args['disabled']}
+		iconAfter={args['iconAfter']}
+		iconBefore={args['iconBefore']}
+		noneText={args['noneText']}
+		onChange={action('onChange')}
+		onClose={action('onClose')}
+		onOpen={action('onOpen')}
+		title={args['title']}
+		placeholder={args['placeholder']}
```
Comment (alexandrumorariu) [inline samples/sampler/stories/default/ExpandableInput.js:32]: Order porps

File: samples/sampler/stories/default/ExpandableList.js [hunk 7]
```diff
@@ -1,35 +1,45 @@
...
+};
+
+export const _ExpandableList = (args) => (
+	<ExpandableList
+		closeOnSelect={args['closeOnSelect']}
+		disabled={args['disabled']}
+		noAutoClose={args['noAutoClose']}
+		noLockBottom={args['noLockBottom']}
+		noneText={args['noneText']}
+		onSelect={action('onSelect')}
```
Comment (alexandrumorariu) [inline samples/sampler/stories/default/ExpandableList.js]: Order props

File: samples/sampler/stories/default/FormCheckboxItem.js [hunk 8]
```diff
@@ -1,43 +1,50 @@
...
+};
+
+export const _FormCheckboxItem = (args) => {
+	const iconPosition = args['iconPosition'];
+	const icon = args['itemIcon'];
+	const itemIcon = nullify(icon ? <Icon>{icon}</Icon> : null);
+	const itemIconPosition = args['itemIconPosition'];
+	return (
+		<FormCheckboxItem
+			disabled={boolean('disabled', Config)}
```
Comment (alexandrumorariu) [inline samples/sampler/stories/default/FormCheckboxItem.js]: You forgot to change both booleans to controls

File: samples/sampler/stories/default/Header.js [hunk 9]
```diff
@@ -1,74 +1,89 @@
...
+export const _Header = (args, context) => {
+	context.noHeader = true;
+
+	const headerInput = args['headerInput'] ? <Input placeholder="placeholder text" /> : null;
+	const childrenSelection = args['children'];
+	const children = prop.children[childrenSelection];
+
+	const story = (
+		<Header
+			title={args['title']}
```
Comment (alexandrumorariu) [inline samples/sampler/stories/default/Header.js:48]: Order props here

File: samples/sampler/stories/default/Header.js [hunk 10]
```diff
@@ -1,74 +1,89 @@
...
+	return story;
+};
+
+boolean('centered', _Header, Config);
+boolean('fullBleed', _Header, Config);
+boolean('headerInput', _Header, Config);
+boolean('hideLine', _Header, Config);
+boolean('noCloseButton', _Header, {displayName: 'Panels'});
+select('controls', _Header, ['no buttons', '1 button', '2 buttons'], {displayName: 'Panels'});
+select('children', _Header, ['no buttons', '1 button', '2 buttons'], Config);
```
Comment (alexandrumorariu) [inline samples/sampler/stories/default/Header.js:75]: Order selects from above

File: samples/sampler/stories/default/IconButton.js [hunk 11]
```diff
@@ -1,56 +1,70 @@
...
+	const iconType = args['icon type'];
+	let children;
+	switch (iconType) {
+		case 'glyph': children = args['icon']; break;
+		case 'url src': children = args['src']; break;
+		default: children = args['custom icon'];
+	}
+	return (
+		<IconButton
+			onClick={action('onClick')}
```
Comment (alexandrumorariu) [inline samples/sampler/stories/default/IconButton.js]: Order props here

File: samples/sampler/stories/default/Image.js [hunk 12]
```diff
@@ -1,54 +1,59 @@
...
+Image.displayName = 'Image';
+
+export default {
+	title: 'Moonstone/Image',
+	component: 'Image'
+};
+
+export const _Image = (args) => (
+	<Image
+		src={args['src']}
```
Comment (alexandrumorariu) [inline samples/sampler/stories/default/Image.js]: Order props here

**Review by alexandrumorariu — APPROVED** [review]
LGTM

File: samples/sampler/stories/util/icons.js [hunk 13]
```diff
@@ -0,0 +1,141 @@
+// Icons For Samples
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/util/icons.js]: please rename the folder of this file to "helper" , just like in Sansdtone. We need consistency
and fix the imports in the stories afterwards

File: samples/sampler/stories/default/IconButton.js [hunk 14]
```diff
@@ -1,56 +1,69 @@
...
-// 		}
-// 	);
+import Button, {ButtonBase} from '@enact/moonstone/Button';
+import IconButton, {IconButtonBase} from '@enact/moonstone/IconButton';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, select, text} from '@enact/storybook-utils/addons/controls';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
+import UIButton, {ButtonBase as UIButtonBase} from '@enact/ui/Button';
+
+import icons from '../util/icons';
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/IconButton.js]: please put this import after the logo import

File: samples/sampler/stories/default/Image.js [hunk 15]
```diff
@@ -1,54 +1,58 @@
...
-// 		{
-// 			info: {
-// 				text: 'The basic Image'
-// 			}
-// 		}
-// 	);
+import Image, {ImageBase, ImageDecorator} from '@enact/moonstone/Image';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {select} from '@enact/storybook-utils/addons/controls';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/Image.js]: this import should be on line 2

File: samples/sampler/stories/default/IncrementSlider.js [hunk 16]
```diff
@@ -1,53 +1,68 @@
...
-// 		{
-// 			info: {
-// 				text: 'Basic usage of IncrementSlider'
-// 			}
-// 		}
-// 	);
+import IncrementSlider, {IncrementSliderBase, IncrementSliderTooltip} from '@enact/moonstone/IncrementSlider';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, number, range, select} from '@enact/storybook-utils/addons/controls';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/IncrementSlider.js]: same here. please check other cases

File: samples/sampler/stories/default/Input.js [hunk 17]
```diff
@@ -1,43 +1,55 @@
...
-// 		{
-// 			info: {
-// 				text: 'The basic Input'
-// 			}
-// 		}
-// 	);
+import Input, {InputBase} from '@enact/moonstone/Input';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, select, text} from '@enact/storybook-utils/addons/controls';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/Input.js]: same here

File: samples/sampler/stories/default/Item.js [hunk 18]
```diff
@@ -1,27 +1,32 @@
...
-// 		),
-// 		{
-// 			info: {
-// 				text: 'Basic usage of Item'
-// 			}
-// 		}
-// 	);
+import Item, {ItemBase} from '@enact/moonstone/Item';
+import {boolean, text} from '@enact/storybook-utils/addons/controls';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/Item.js:2]: same here

File: samples/sampler/stories/default/Button.js [hunk 19]
```diff
@@ -1,44 +1,55 @@
...
-// 		{
-// 			info: {
-// 				text: 'The basic Button'
-// 			}
-// 		}
-// 	);
+import Button, {ButtonBase} from '@enact/moonstone/Button';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, select, text} from '@enact/storybook-utils/addons/controls';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/Button.js]: this import should be on line 2

File: samples/sampler/stories/default/CheckboxItem.js [hunk 20]
```diff
@@ -1,44 +1,51 @@
...
-// 			}
-// 		}
-// 	);
+import CheckboxItem from '@enact/moonstone/CheckboxItem';
+import Icon from '@enact/moonstone/Icon';
+import Item, {ItemBase} from '@enact/moonstone/Item';
+import ToggleItem from '@enact/moonstone/ToggleItem';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, select, text} from '@enact/storybook-utils/addons/controls';
+import {mergeComponentMetadata, nullify} from '@enact/storybook-utils';
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/CheckboxItem.js]: this import should be on line 5

File: samples/sampler/stories/default/ContextualPopupDecorator.js [hunk 21]
```diff
@@ -1,53 +1,63 @@
...
-// 				text: 'Basic usage of ContextualPopupDecorator'
-// 			}
-// 		}
-// 	);
+import BodyText from '@enact/moonstone/BodyText';
+import Button from '@enact/moonstone/Button';
+import {ContextualPopupDecorator} from '@enact/moonstone/ContextualPopupDecorator';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, select, text} from '@enact/storybook-utils/addons/controls';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/ContextualPopupDecorator.js]: this import should be on line 4

File: samples/sampler/stories/default/DatePicker.js [hunk 22]
```diff
@@ -1,38 +1,50 @@
...
-// 		{
-// 			info: {
-// 				text: 'The basic DatePicker'
-// 			}
-// 		}
-// 	);
+import DatePicker, {DatePickerBase} from '@enact/moonstone/DatePicker';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, text} from '@enact/storybook-utils/addons/controls';
+import {mergeComponentMetadata, removeProps} from '@enact/storybook-utils';
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/DatePicker.js]: this import should be on line 2

File: samples/sampler/stories/default/DaySelector.js [hunk 23]
```diff
@@ -1,33 +1,37 @@
...
-// 		{
-// 			info: {
-// 				text: 'Basic usage of DaySelector'
-// 			}
-// 		}
-// 	);
+import DaySelector, {DaySelectorBase} from '@enact/moonstone/DaySelector';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, select} from '@enact/storybook-utils/addons/controls';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/DaySelector.js]: this import should be on line 2

File: samples/sampler/stories/default/Dialog.js [hunk 24]
```diff
@@ -1,49 +1,58 @@
...
-// 			}
-// 		}
-// 	);
+import BodyText from '@enact/moonstone/BodyText';
+import Button from '@enact/moonstone/Button';
+import Dialog from '@enact/moonstone/Dialog';
+import Popup from '@enact/moonstone/Popup';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, text} from '@enact/storybook-utils/addons/controls';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/Dialog.js]: this import should be on line 5

File: samples/sampler/stories/default/Dropdown.js [hunk 25]
```diff
@@ -1,40 +1,47 @@
...
-// 			info: {
-// 				text: 'A quick, inline, value-selection component'
-// 			}
-// 		}
-// 	);
+import Button, {ButtonBase} from '@enact/moonstone/Button';
+import Dropdown, {DropdownBase} from '@enact/moonstone/Dropdown';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, range, select, text} from '@enact/storybook-utils/addons/controls';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/Dropdown.js]: this import should be on line 3

File: samples/sampler/stories/default/EditableIntegerPicker.js [hunk 26]
```diff
@@ -1,46 +1,58 @@
...
-// 		{
-// 			info: {
-// 				text: 'Basic usage of EditableIntegerPicker'
-// 			}
-// 		}
-// 	);
+import EditableIntegerPicker, {EditableIntegerPickerBase} from '@enact/moonstone/EditableIntegerPicker';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, number, select, text} from '@enact/storybook-utils/addons/controls';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/EditableIntegerPicker.js]: this import should be on line 2

File: samples/sampler/stories/default/ExpandableInput.js [hunk 27]
```diff
@@ -1,42 +1,51 @@
...
-// 		{
-// 			info: {
-// 				text: 'Basic usage of ExpandableInput'
-// 			}
-// 		}
-// 	);
+import ExpandableInput, {ExpandableInputBase} from '@enact/moonstone/ExpandableInput';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, select, text} from '@enact/storybook-utils/addons/controls';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/ExpandableInput.js]: this import should be on line 2

File: samples/sampler/stories/default/ExpandableItem.js [hunk 28]
```diff
@@ -1,43 +1,51 @@
...
-// 				text: 'Basic usage of ExpandableItem'
-// 			}
-// 		}
-// 	);
+import ExpandableItem, {Expandable, ExpandableItemBase} from '@enact/moonstone/ExpandableItem';
+import Icon from '@enact/moonstone/Icon';
+import Item from '@enact/moonstone/Item';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, select, text} from '@enact/storybook-utils/addons/controls';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/ExpandableItem.js]: this import should be on line 3

File: samples/sampler/stories/default/ExpandableList.js [hunk 29]
```diff
@@ -1,35 +1,44 @@
...
-// 		{
-// 			info: {
-// 				text: 'Basic usage of ExpandableList'
-// 			}
-// 		}
-// 	);
+import ExpandableList, {ExpandableListBase} from '@enact/moonstone/ExpandableList';
+import {action} from '@enact/storybook-utils/addons/actions';
+import {boolean, select, text} from '@enact/storybook-utils/addons/controls';
+import {mergeComponentMetadata} from '@enact/storybook-utils';
```
Comment (daniel-stoian-lgp) [inline samples/sampler/stories/default/ExpandableList.js]: this import should be on line 2

**Comment by daniel-stoian-lgp** [comment]
please sort all mergeComponentMetadata imports

**Review by daniel-stoian-lgp — APPROVED** [review]

**Pull request merged**